new-games-to-db/main.py

```
@@ -80,7 +80,7 @@ def get_year_from_db_maybe_update(db: firestore.Client, day: int, dry_run: bool)
   return pc_year
   
 
-def new_games_to_db(request):
+def new_games_to_db(args=[]):
```

Ruff (0.11.9)

`new_games_to_db` is too complex (17 > 10)

(C901)